.github/workflows/ci.yml

```diff
@@ -57,6 +57,8 @@ jobs:
         include:
           - os: ubuntu-latest
             e2e: true
+          - os: macos-latest
```

Comment:
Simpler would be to explicitly *not* run it on windows-latest

Reply:
Agreed but I didn't want to mess with the config too much.

Ideally, I would run them on windows too but for that we need to change how we download the blockchain nodes.